ui/handler.go

```diff
@@ -29,3 +30,7 @@ func (h *handler) HealthCheck() Health {
 func (h *handler) Example() Example {
 	return NewUser(h.uc.Example(), h.logger)
 }
+
+func (h *handler) User() User {
+	return NewUsers(h.uc.User(), h.logger)
```

Comment:
[imo] 下記の方が良いね！
Example邪魔だと思うから消して良いよ！
```suggestion
	return NewUser(h.uc.User(), h.logger)
```